### ISSUE TITLE
Support Python 3.9 with Kodi 19

### ISSUE THREAD

**Issue by nylocx** [herrnst/script.xbmc.lcdproc#61]
If I try to run LCDproc with Kodi 19 and Python 3.9 I get the following error:

```
2021-04-25 11:27:46.591 T:2591    ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'AttributeError'>
                                                   Error Contents: 'ElementTree' object has no attribute 'getiterator'
                                                   Traceback (most recent call last):
                                                     File "/home/alex/.kodi/addons/script.xbmc.lcdproc/main.py", line 33, in <module>
                                                       xbmclcd.RunLCD()
                                                     File "/home/alex/.kodi/addons/script.xbmc.lcdproc/resources/lib/xbmclcdproc.py", line 97, in RunLCD
                                                       if self.HandleConnectLCD():
                                                     File "/home/alex/.kodi/addons/script.xbmc.lcdproc/resources/lib/xbmclcdproc.py", line 86, in HandleConnectLCD
                                                       ret = self._LCDproc.Initialize()
                                                     File "/home/alex/.kodi/addons/script.xbmc.lcdproc/resources/lib/lcdproc.py", line 200, in Initialize
                                                       if LcdBase.Initialize(self):
                                                     File "/home/alex/.kodi/addons/script.xbmc.lcdproc/resources/lib/lcdbase.py", line 204, in Initialize
                                                       bGotDefaultSkin = self.LoadSkin(__lcddefaultxml__, True)
                                                     File "/home/alex/.kodi/addons/script.xbmc.lcdproc/resources/lib/lcdbase.py", line 250, in LoadSkin
                                                       for element in doc.getiterator():
                                                   AttributeError: 'ElementTree' object has no attribute 'getiterator'
                                                   -->End of Python script error report<--
                                                   
2021-04-25 11:27:46.603 T:2591     INFO <general>: Python interpreter stopped
```

getiterator was deprecated a while back and is removed from 3.9.

Kind regards,
Alex

**Comment by nylocx** [comment]
Ok, I see this should be solved in the current master as of commit 594f678.
So we just need a newer version in the repository.

**Comment by herrnst** [comment]
script.xbmc.lcdproc-3.90.2 submission pending: https://github.com/xbmc/repo-scripts/pull/2009
